main-app/app/index.html

```diff
@@ -34,26 +36,22 @@
         </table>
     </main>
 
-
     <aside>
         <div class="playerControls">
-            <button class="newGame" ng-click="newGame()">New Game</button>
+            <img  class="newGameButton" src="images/newGame.png" alt="New Game" ng-click="newGame()">
 
-            <div class="playerSwitcherTable">
+            <div class="playerSwitchTable">
```

Comment:
Don't use tables for styling - only use for tables of data, e.g. you could keep stats of player1 wins vs player 2 wins in a table if you wanted. - Use divs.